stable/buzzfeed-sso/templates/ingress.yaml

```diff
@@ -0,0 +1,46 @@
+{{- $fullName := include "buzzfeed-sso.fullname" . -}}
+{{- $authDomain := .Values.auth.domain -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "buzzfeed-sso.name" . }}
+    chart: {{ template "buzzfeed-sso.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+{{- with .Values.ingress.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+{{- end }}
+spec:
+  tls:
+    - hosts:
+        - {{ $authDomain }}
+      secretName: sso-auth-tls-secret
```

Comment:
This secret has to be set from `values`, hasn't it?

Reply:
I guess we can make it dynamic, though an auth domain is always required. Do you think is essential for this PR or something that can be implemented later?

Reply:
In my case TLS secret is same for auth and proxy hosts and it already exists, so when working with predefined secrets, this line should be dynamic.
In my copy I added `tls.secretName` for auth to the list of hosts in ingress, but excluded auth domain from the loop in configmap template in order to not have it in `upstream_configs.yml`. This eliminates the necessity of another value for just auth secret and make values file more consistent.

Reply:
I have added a `tls.secretName` configuration option, so the secret name can be dynamically changed